h/schemas/admin_group.py

```diff
@@ -47,8 +53,22 @@ def validate(node, value):
     return validate
 
 
+@colander.deferred
+def group_organization_select_widget(node, kw):
+    orgs = kw['organizations']
+    org_names = ['{} ({})'.format(org.name, org.authority) for org in orgs]
```

Comment:
Glad to see you display the authority here! Very helpful (arguably totally necessary, even).